src/app/api-server/src/app.rs

```diff
@@ -244,6 +244,8 @@ pub async fn init_dependencies(
     local_dir: &Path,
     e2e_http_port: Option<u16>,
 ) -> Result<CatalogBuilder, InternalError> {
+    println!("!!! 1 {config:?}");
```

Comment:
oopsie

Reply:
damn... my NOT best practices for debugging E2E should have been kept secret

Fixed